form-update.rst

```diff
@@ -0,0 +1,140 @@
+Updating forms
+=================
+
+.. _change-existing:
+
+Making changes to existing form
+---------------------------------
+
+Certain changes which don't involve adding or removing a question can be made without needing to replace the existing forms. The kind of changes are:
+
+- Changing the text or translation of a label
+- Changing validations, calculations, relevants
+- Changing options for a select_one or select_multiple
```

Comment:
Is that about external csv files too or only about options that we store in a form? Because we can edit csv files too like media files.

Reply:
I checked this and we can update the options in external csv files too but in that case, we don't need to update the version number.